docs/.c4/views/functional-layers/layers.views.c4

```diff
@@ -0,0 +1,99 @@
+views "uds-core-functional-layers" {
+  // Base Layer - Istio, UDS Operator, Policy Engine
+  view base-layer of cloud.cluster {
+    title "UDS Core Base Layer"
+
+    include
+      cloud,
+      cluster,
+      uds-core,
+      istio-system.*,
+      istio-tenant-gateway.*,
+      istio-admin-gateway.*,
+      pepr-system.*,
+      zarf.*,
+      kubernetes-control-plane.*
```

Comment:
These aren't part of our base layer, although they are relevant for sure... I think it might be confusing adding them to the base diagram though?